store/tikv/async_commit_fail_test.go

```diff
@@ -36,7 +36,7 @@ type testAsyncCommitFailSuite struct {
 var _ = SerialSuites(&testAsyncCommitFailSuite{})
 
 func (s *testAsyncCommitFailSuite) SetUpTest(c *C) {
-	s.testAsyncCommitCommon.setUpTest(c)
```

Comment:
Why we do not use tikv to run the test?

Reply:
Not all tests of async commit can run with tikv. I'm going to adjust them later.